subgraphs/opyn-gamma/src/entities/snapshot.ts

```diff
@@ -0,0 +1,638 @@
+    pool.cumulativeVolumeUSD;
+  snapshot.cumulativeVolumeByTokenAmount = snapshot.dailyVolumeByTokenAmount =
+    pool.cumulativeVolumeByTokenAmount;
+  snapshot.cumulativeVolumeByTokenUSD = snapshot.dailyVolumeByTokenUSD =
+    pool.cumulativeVolumeByTokenUSD;
+
+  snapshot.cumulativeDepositedVolumeByTokenAmount =
+    snapshot.dailyDepositedVolumeByTokenAmount =
+      pool.cumulativeDepositedVolumeByTokenAmount;
+  snapshot.cumulativeDepositedVolumeByTokenUSD =
+    snapshot.dailyDepositedVolumeByTokenUSD =
+      pool.cumulativeDepositedVolumeByTokenUSD;
+  snapshot.cumulativeDepositedVolumeUSD = snapshot.dailyDepositedVolumeUSD =
+    pool.cumulativeDepositedVolumeUSD;
+
+  snapshot.cumulativeWithdrawVolumeUSD = snapshot.dailyWithdrawVolumeUSD =
+    pool.cumulativeWithdrawVolumeUSD;
+  snapshot.cumulativeWithdrawVolumeByTokenAmount =
+    snapshot.dailyWithdrawVolumeByTokenAmount =
+      pool.cumulativeWithdrawVolumeByTokenAmount;
+  snapshot.cumulativeWithdrawVolumeByTokenUSD =
+    snapshot.dailyWithdrawVolumeByTokenUSD =
+      pool.cumulativeWithdrawVolumeByTokenUSD;
+
+  snapshot.cumulativeClosedVolumeUSD = snapshot.dailyClosedVolumeUSD =
+    pool.cumulativeClosedVolumeUSD;
+  snapshot.cumulativeExerciseVolumeUSD = snapshot.dailyExerciseVolumeUSD =
+    pool.cumulativeExercisedVolumeUSD;
+
+  const prevSnapshotDays =
+    pool._lastDailySnapshotTimestamp.toI32() / SECONDS_PER_DAY;
+  const prevSnapshot = LiquidityPoolDailySnapshot.load(
+    pool.id.concatI32(prevSnapshotDays)
+  );
+  if (prevSnapshot) {
+    snapshot.dailySupplySideRevenueUSD =
+      pool.cumulativeSupplySideRevenueUSD.minus(
+        prevSnapshot.cumulativeSupplySideRevenueUSD
+      );
+    snapshot.dailyProtocolSideRevenueUSD =
+      pool.cumulativeProtocolSideRevenueUSD.minus(
+        prevSnapshot.cumulativeProtocolSideRevenueUSD
+      );
+    snapshot.dailyTotalRevenueUSD = pool.cumulativeTotalRevenueUSD.minus(
+      prevSnapshot.cumulativeTotalRevenueUSD
+    );
+    snapshot.dailyEntryPremiumUSD = pool.cumulativeEntryPremiumUSD.minus(
+      prevSnapshot.cumulativeEntryPremiumUSD
+    );
+    snapshot.dailyExitPremiumUSD = pool.cumulativeExitPremiumUSD.minus(
+      prevSnapshot.cumulativeExitPremiumUSD
+    );
+    snapshot.dailyTotalPremiumUSD = pool.cumulativeTotalPremiumUSD.minus(
+      prevSnapshot.cumulativeTotalPremiumUSD
+    );
+    snapshot.dailyDepositPremiumUSD = pool.cumulativeDepositPremiumUSD.minus(
+      prevSnapshot.cumulativeDepositPremiumUSD
+    );
+    snapshot.dailyWithdrawPremiumUSD = pool.cumulativeWithdrawPremiumUSD.minus(
+      prevSnapshot.cumulativeWithdrawPremiumUSD
+    );
+    snapshot.dailyTotalLiquidityPremiumUSD =
+      pool.cumulativeTotalLiquidityPremiumUSD.minus(
+        prevSnapshot.cumulativeTotalLiquidityPremiumUSD
+      );
+    snapshot.dailyCollateralVolumeUSD =
+      snapshot.cumulativeCollateralVolumeUSD!.minus(
+        prevSnapshot.cumulativeCollateralVolumeUSD!
```

Comment:
I don't know for sure, but I think the runtime error may be caused because we are always assuming this field is populated when it may be null

Reply:
The runtime error was already fixed, I just forgot to deploy the other network deployments.
But I ended up removing that field anyway for now.